File: programmable_video/ios/Classes/AudioSettings.swift
internal class AudioSettings: NSObject {
    var speakerEnabled: Bool = true
    var bluetoothPreferred: Bool = true

    init(speakerEnabled: Bool = true, bluetoothEnabled: Bool = true) {
        self.speakerEnabled = speakerEnabled
        self.bluetoothPreferred = bluetoothEnabled
    }

    func reset() {
        self.speakerEnabled = true
        self.bluetoothPreferred = true
    }
}

File: programmable_video/android/src/main/kotlin/twilio/flutter/twilio_programmable_video/AudioSettings.kt
package twilio.flutter.twilio_programmable_video

class AudioSettings(speakerEnabled: Boolean = true, bluetoothPreferred: Boolean = true) {
    var speakerEnabled: Boolean
    var bluetoothPreferred: Boolean

    init {
        this.speakerEnabled = speakerEnabled
        this.bluetoothPreferred = bluetoothPreferred
    }

    fun reset() {
        speakerEnabled = true
        bluetoothPreferred = true
    }
}
